src/main/java/frc/robot/util/Dashboards.java

```diff
@@ -27,6 +31,26 @@ public static void initVoltageDashboard() {
         }
     }
 
+    public static void initCurrentDashboard(){
+        if (currentNeedsInit){
+            ShuffleboardTab tab = Shuffleboard.getTab("Current Info");
+            //Arm Motors
+            tab.addDouble("Right Arm Motor Current",() -> PDH.getCurrent(18));
+            tab.addDouble("Left Arm Motor Current",() -> PDH.getCurrent(5));
+            //Shooter Motors
+            tab.addDouble("Shooter Motor 1 Current",() -> PDH.getCurrent(6));
```

Comment:
Lets make a TEMPORARY (like, we are going toi delete it before merging this) command for the shooter subsystem that takes in a can id and runs the motor for that can id at like 10%. Then we can watch this dashboard and know whiuch PD channel corresponds to each can ID. Then record that in Constants and delete the temp command.